kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-on-pv/common/utility.sh

```diff
@@ -39,7 +39,32 @@ function checkDomainSecret {
   fi
 }
 
+#
+# traceDirs before|after
+#   Trace contents and owner of DOMAIN_HOME/LOG_HOME/DATA_HOME directories
+#
+function traceDirs() {
+  echo "id = '`id`'"
+  local indir
+  for indir in DOMAIN_HOME_DIR DOMAIN_ROOT_DIR DOMAIN_LOGS_DIR; do
+    [ -z "${!indir}" ] && continue
+    echo "Directory trace for $indir=${!indir} ($1)"
+    local cnt=0
+    local odir=""
+    local cdir="${!indir}/*"
+    while [ ${cnt} -lt 30 ] && [ ! "$cdir" = "$odir" ]; do
```

Comment:
I am not sure I understand what this is meant to do. Can you please give me a quick explanation?

Reply:
It shows the files and their ownership within the given directory, and it shows the ownership of the given directory and each of its parents.   See this pull's comment for sample output.

Reply:
It also shows the current user and the user's gids via the `id` command.

Reply:
ok i got it, thanks